src/index.html

```diff
@@ -4,13 +4,55 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action=""
+      data-qa="big"
+      method="GET"
+      class="big-form-bar"
+    >
+
+      <input
+        type="text"
+        class="bar big-bar"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+
+      <img
+        src="images/Search.svg"
+        alt=""
+        class="image-big"
+      >
+
+    </form>
```

Comment:
не нужно так много отступов
```suggestion
    <form
      action=""
      data-qa="big"
      method="GET"
      class="big-form-bar"
    >
      <input
        type="text"
        class="bar big-bar"
        data-qa="keypress"
        placeholder="Try &quot;Los Angeles&quot;"
      >
      <img
        src="images/Search.svg"
        alt=""
        class="image-big"
      >
    </form>
```